MyStoreAtomation/CatalogTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Infrastructure;
+using FluentAssertions;
+
+//ReferenceTypeAssertions ==> for later assertions
+
+namespace MyStoreAtomation
+{
+    [TestClass]
+    public class CatalogTests : UnitTestBase
+    {
+        [TestMethod]
+        public void AddToCartButtonPopsTest()
+        {
+            var threwExeption = false;
+            try
+            {
+                //hover over item[0], try to access it's addToCart button, if it won't apper it will throw an exception.
+                var button = CatalogPage.Items[0].HoverOnItemByIndex().Items[0].AddToCartButton;
+            }
+
+            catch
+            {
+                threwExeption = true;
+            }
+
+            threwExeption.Should().BeFalse();
+        }
+        
+        [TestMethod]
+        public void AddToCartButtonDisapeersTest()
+        {
+            var threwExeption = false;
+            try
```

Comment:
why not check if the element is visible instead of checking if an exception is being thrown. any exception could be thrown, it is very unreliable way to check if an element disapears